src/lib/logger.ts

```diff
@@ -154,9 +154,11 @@ export class Logger {
 }
 
 function sendToLoggingService(data: LogMessage, user: UserModelType) {
+  const isProduction = user.portal === productionPortal || data.parameters?.portal === productionPortal;
+  const url = logManagerUrl[isProduction ? "production" : "dev"];
   if (DEBUG_LOGGER) {
     // eslint-disable-next-line no-console
-    console.log("Logger#sendToLoggingService sending", data, "to", logManagerUrl);
+    console.log("Logger#sendToLoggingService sending", data, "to", url);
```

Comment:
Does this change make sense?

Seems to me we'd want to include the actual URL used, not all the URLs that could possibly have been used.

Reply:
Yes, I suspect `logManagerUrl` started out as a single value and then was later changed to a map without this log statement being updated.

Reply:
@kswenson Thanks. Did you have a chance to look over the other changes? Do they make sense, too?